src/app/player.component.ts

```diff
@@ -50,19 +77,22 @@ export class PlayerComponent {
       }
       this.rootBlock = this.parserService.parseUnitDefinition(startData.unitDefinition.split(/\r?\n/g));
       this.rootBlock.check(storedResponses);
+      // check if presentationProgress could be sent (e.g. small pages which hasn't to be scrolled)
```

Comment:
Kleiner Rechtschreibfehler. :)

Entweder "small page which hasn't to be scrolled" oder "small pages which haven't to be scrolled"

Reply:
fixed